docs/configuration-reference/platforms/packet.md

```diff
@@ -212,6 +226,9 @@ node_type = var.custom_default_worker_type
 | `certs_validity_period_hours`         | Validity of all the certificates in hours.                                                                                                                                    | 8760            | false    |
 | `worker_pool`                         | Configuration block for worker pools. There can be more than one.                                                                                                             | -               | true     |
 | `worker_pool.count`                   | Number of workers in the worker pool. Can be changed afterwards to add or delete workers.                                                                                     | 1               | true     |
+| `worker_pool.ssh_pubkeys`             | Optional list of SSH public keys for user `core`. If not provided then SSH public keys provided for controllers will be used for the worker pool.                             | []              | false    |
```

Comment:
```suggestion
| `worker_pool.ssh_pubkeys`             | Optional list of SSH public keys for user `core`. If not provided, the value of `ssh_pubkeys` is used.                             | []              | false    |
```

- Omit "then" after if.
- Reference the concrete knob whose value is used.
- Avoid future tense.

If you change this, please fix all similar occurrences.